src/functionaltests/java/com/ericsson/ei/subscriptions/trigger/SubscriptionTriggerSteps.java

```diff
@@ -214,6 +220,35 @@ private void validateSubscriptionsSuccessfullyAdded(String endPoint) throws Exce
         }
     }
 
+    /**
+     * Checks that an enpoint got at least the number of calls as expected.
+     * 
+     * @param endpoints
+     *            List of endpoints to check.
+     * @param expectedCalls
+     *            Integer with the least number of calls.
+     * @return true if all endpoints had atleast the number of calls as
+     *         expected.
+     * @throws JSONException
+     * @throws InterruptedException
+     */
+    private boolean allEndpointsGotAtLeastXCalls(final List<String> endpoints, int expectedCalls)
+            throws JSONException, InterruptedException {
```

Comment:
Where is the unit tests for this function?

Reply:
I guess no unit tests has been created for the other functions. So I guess I can approve this change for the moment.
But we should have unit test implemented for the logics in these test base class and functions in a near future.